Dockerfile-GPU

```diff
@@ -1,11 +1,33 @@
-FROM nvidia/cuda:10.2-base-ubuntu18.04
+#FROM nvidia/cuda:10.2-base-ubuntu18.04
+FROM pytorch/pytorch:1.5-cuda10.1-cudnn7-devel
+
 RUN apt-get update && apt-get install -y python3-pip && apt-get install -y python3-pip && apt-get install -y git
```

Comment:
I think Python install might not be required here anymore as PyTorch image should have Python.

Reply:
Yep, good point